src/data/python-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const pythonQuiz = [
+   {
+    Question: 
+      "Which mode create new file if the file does not exist?",
+    Answer: "Both of the above",
+    Distractor1: "Write mode",
+    Distractor2: "Append mode",
+    Distractor3: "Both of the above",
+    Explanation: 
+      "In Python file handling,To create new file in python Write or Append parameter will be called.",
```

Comment:
let's go with this rewrite of the question to focus on the write method so the question and multiple choice answers are better tied together 👍  

```suggestion
    Question: 
      "In Python, which method is used to write text to a file?",
    Answer: "write()",
    Distractor1: "read()",
    Distractor2: "appending()",
    Distractor3: "writing()",
    Explanation: 
      "In Python the write() method is used to write text to a file.",
```